tests/pxScene2d/testRunner/tests.json

```diff
@@ -11,7 +11,7 @@
       {"url":"../tests/test_getObjectById.js", "title":"test_getObjectById","useBaseURI":"true"},
       {"url":"../tests/test_fancyNewAnimate.js", "title":"fancyNewAnimate","useBaseURI":"true","timeToRun":"10000"},
       {"url":"../tests/test_pxConstants.js", "title":"test_pxConstants","useBaseURI":"true"},
-      {"url":"../tests/test_pxImageA.js", "title":"test_pxImageA","useBaseURI":"true"},
+      {"url":"../tests/test_pxImageA_3.js", "title":"test_pxImageA_3", "useBaseURI":"true"},
```

Comment:
Please call this new test "test_pxImageA_v2.js" in order to not confuse with the different test that is "test_pxImageA_2.js"